ankihub/main/note_type_management.py

```diff
@@ -106,5 +106,14 @@ def note_types_with_template_changes_for_deck(ah_did: uuid.UUID) -> List[Notetyp
     return ids
 
 
-def update_deck_templates(ah_did: uuid.UUID, note_type: NotetypeDict) -> None:
-    print("update_deck_templates", ah_did)
+def update_deck_templates(ah_did: uuid.UUID, note_type: NotetypeDict) -> NotetypeDict:
+    client = AddonAnkiHubClient()
+    db_note_type = ankihub_db.note_type_dict(ah_did, note_type["id"])
+
+    db_note_type["tmpls"] = note_type["tmpls"]
```

Comment:
:wrench: We allow users to add custom content to the end of the note type templates, after the "ankihub end" comment. We should remove this comment and custom content before sending the note templates to the API  and storing them in the DB.
```suggestion
    note_type = note_type_with_ankihub_end_comment_removed(note_type)
    db_note_type["tmpls"] = note_type["tmpls"]
```